pkg/sql/plan/build_ddl.go

```diff
@@ -1376,15 +1376,16 @@ func buildSecondaryIndexDef(createTable *plan.CreateTable, indexInfos []*tree.In
 				return moerr.NewNotSupported(ctx.GetContext(), fmt.Sprintf("VECTOR column '%s' cannot be in index", name))
 			}
 
-			if strings.Compare(name, pkeyName) == 0 {
-				isPkAlreadyPresentInIndexParts = true
-			}
+			//if strings.Compare(name, pkeyName) == 0 {
+			//	isPkAlreadyPresentInIndexParts = true
+			//}
 			indexParts = append(indexParts, name)
 		}
 
-		if !isPkAlreadyPresentInIndexParts {
-			indexParts = append(indexParts, pkeyName)
-		}
+		//if !isPkAlreadyPresentInIndexParts {
+		// TODO: see how to do aliasing for the pkeyName @qingx
+		indexParts = append(indexParts, catalog.IndexTablePrimaryColNameSK)
```

Comment:
We were previously adding PKName in the index parts.

According to the new suggestion, we would need to find a way to use a different name, which points to `originalTable.PKName`. I am not sure how to do it here. 

My understanding is that indexParts should map to the defined columns inside the Original Table.